packages/core-ethereum/src/channel/index.ts

```diff
@@ -203,11 +203,8 @@ class ChannelFactory {
       }
     )
 
-    await sign(await ticket.hash, this.coreConnector.account.keys.onChain.privKey, undefined, {
-      bytes: signedTicket.buffer,
-      offset: signedTicket.signatureOffset
-    })
-
+    const signature = await sign(await ticket.hash, this.coreConnector.account.keys.onChain.privKey)
+    signedTicket.set(signature, signedTicket.signatureOffset - signedTicket.byteOffset)
```

Comment:
It could make sense to check the length before writing the signature

Reply:
this will be resolved in a future PR.